.github/workflows/kubernetes-integration-test.yaml

```diff
@@ -0,0 +1,49 @@
+name: kubernetes-integration-test
+on: [push]
+jobs:
+  kubernetes-integration-test:
+    runs-on: ubuntu-latest
+    steps:
+      - 
+        name: Checkout code
+        uses: actions/checkout@v2
+      - 
+        name: Go build
+        run: CGO_ENABLED=0 GOOS=linux go build .
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      -
+        name: Docker build
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          load: true
+          tags: thegreenwebfoundation/grid-intensity-exporter:integration-test
+      - 
+        name: Create kubernetes Kind Cluster
+        uses: helm/kind-action@v1.1.0
+      - 
+        name: Load image into Kind cluster
+        run: kind load docker-image --name chart-testing thegreenwebfoundation/grid-intensity-exporter:integration-test
+      - 
+        name: Set up Helm
+        uses: azure/setup-helm@v1
+      - 
+        name: Helm install
+        run: helm install grid-intensity-exporter helm/grid-intensity-exporter --set image.tag=integration-test
+      - 
+        name: Set up Kubectl
+        uses: azure/setup-kubectl@v1
+      - 
+        name: Wait for ready pod
+        run: kubectl wait --for=condition=ready -l app.kubernetes.io/name=grid-intensity-exporter pod
+      - 
+        name: Create port forward to metrics port
+        run: kubectl port-forward deployment/grid-intensity-exporter 8000:8000 &
```

Comment:
We need to wait for the pod to be ready and create a port forward connection to access the metrics.

Within the cluster other components can access the exporter via its service.